### DISCUSSION TITLE
Copy only selected/filtered/different files left to right?

### DISCUSSION THREAD

**Discussion by TonyGravagno** [WinMerge/winmerge#2475] [Q&A] [answered]
I have code projects that include binaries, folders like .git, node_modules, etc. A .flt filters these out. When I use the context menu to Copy files, I don't want to copy "skipped files" that have already been filtered out. But the Copy selects and copies everything, including skipped files.

This can be a bug or a feature. I think the decision should be added to the confirmation prompt.

This becomes important with delete operations too. We might want to filter all files that have changed and then delete them. It would be tragic if Delete also deletes everything, even skipped files.

I'm still using 2.16.32 ... I'm sorry if this was changed since then.

**Comment by sdottaka** [comment, marked as answer]
This behavior has been reported a few times, and we'd like to improve it, but haven't gotten around to it yet.
Related issues: https://github.com/WinMerge/winmerge/issues/198, https://github.com/WinMerge/winmerge/issues/1168, https://github.com/WinMerge/winmerge/issues/1350, https://github.com/WinMerge/winmerge/issues/1995

**Reply by TonyGravagno** [reply]
Got it. Thank you VERY much. Sorry I missed the tickets ... I did look.  🙈
